tools/c7n_salactus/c7n_salactus/inventory.py

```diff
@@ -62,7 +62,20 @@ def load_manifest_file(client, bucket, schema, versioned, ifilters, key_info):
             yield keys
 
 
+def is_delete_marker(ischema, kr):
+    """Is the given key record for a delete marker?
+        "fileSchema" : "Bucket, Key, VersionId, IsLatest,
+                        IsDeleteMarker, Size, LastModifiedDate, EncryptionStatus"
+    """
+    if 'IsDeleteMarker' not in ischema:
+        return False
+    return kr[ischema['IsDeleteMarker']] == 'true'
```

Comment:
Is it OK to do this for every record?

Reply:
ischema usage here assumes a dict, on a versioned bucket inventory requires a little verification as well as a change of use in this pr if ischema is a list. also given the volume i'd skip the function call and inline this usage.